<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dSettings</class>
 <widget class="QDialog" name="dSettings">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbl_file">
         <property name="text">
          <string>&amp;Path to db</string>
         </property>
         <property name="buddy">
          <cstring>lne_path</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lne_path"/>
       </item>
       <item>
        <widget class="QToolButton" name="tb_patch">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="chb_splash">
         <property name="text">
          <string>&amp;Splash enable</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btn_box">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lne_path</tabstop>
  <tabstop>tb_patch</tabstop>
  <tabstop>chb_splash</tabstop>
  <tabstop>btn_box</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
